packages/desktop/components/popups/BalanceFinderPopup.svelte

```diff
@@ -34,19 +45,25 @@
     export let title: string
     export let body: string
 
-    const { isStrongholdLocked, network } = $activeProfile
-    const searchAlgorithm: SearchAlgorithmType =
-        network.id === NetworkId.Iota || NetworkId.IotaAlphanet ? SearchAlgorithmType.IDS : SearchAlgorithmType.BFS
-
     let error = ''
     let isBusy = false
+    let searchInCurrentWallet: boolean = false
+    let shouldInitSearch: boolean = false
```

Comment:
These 2 values are not going to persist if the popup remounts after asking for the stronghold password 🙏🏼